app/src/main/java/com/fastaccess/ui/modules/profile/starred/ProfileStarredPresenter.java

```diff
@@ -69,8 +69,9 @@
         if (starredCount == -1) {
             observable = Observable.zip(RestProvider.getUserService(isEnterprise()).getStarred(parameter, page),
                     RestProvider.getUserService(isEnterprise()).getStarredCount(parameter), (repoPageable, count) -> {
-                        if (count != null) {
-                            starredCount = count.getLast();
+                        if (repoPageable != null
+                                && repoPageable.getItems() != null) {
+                            starredCount = repoPageable.getItems().size();
```

Comment:
This won't work, im using search api to get only one page contains 1 item in the list so I could get the count from the API this will result into only counting 1 (the list) and the whole repos count.